cfme/tests/configure/test_visual_infra.py

```diff
@@ -214,27 +209,27 @@ def test_start_page(request, setup_a_provider, start_page):
     # assert nav.is_page_active(*steps) or nav.is_page_active(*longer_steps), "Landing Page Failed"
 
 
-def test_infraprovider_noquads(request, setup_a_provider, set_infra_provider_quad):
-    navigate_to(setup_a_provider, 'All')
+def test_infraprovider_noquads(request, infra_provider, set_infra_provider_quad):
```

Comment:
Have you tried just importing the base Infra class (or even one of the type specific classes) and navigating against it instead? The 'All' destination shouldn't require an object